app/models/spree/product.rb

```diff
@@ -274,7 +274,7 @@ def deleted?
     end
 
     def available?
```

Comment:
any idea where available is used?

Reply:
Ah, interesting! Maybe it's dead code?

I just noticed this quick bugfix in the `2-2-stable` branch so I made this PR as a reminder to look at it at some point, but maybe we don't even use this method at all... I guess we could try deleting the method and specs instead?

Reply:
> I guess we could try deleting the method and specs instead?

yeah, we can try that